synonymdev/bitkit-react-native
# chore(lang): Update copy after review by PM

### Description
Closes #1689

Updates various copy where improvements were deemed necessary by PM team.

Also applied a few cleanups and code optimisations related only to copy, either fullscreen error screen or toasts, and extracted some strings to i18n files to make them translateable.

### Linked Issues/Tasks
- [Update error and toast msg in Bitkit](https://app.asana.com/0/1189635884316057/1207412676031883/f)

### Type of change


### QA Notes

Check `Bitkit error/success messages` sheet for references between updated copy and impacted flows.
